modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaFDSDataProviderCheck.java

```diff
@@ -33,6 +34,14 @@ protected String format(
 		List<String> implementedClassNames =
 			javaClass.getImplementedClassNames();
 
+		if (implementedClassNames.contains("FDSDataProvider")) {
+			content = _updateProviderKey(content);
+
+			content = _updateImplementedClass(content);
```

Comment:
If the UpgradeJavaCheck has already been executed, the class will be like this:

Before
```
 public class UpgradeJavaClayDataSetDataProviderCheck implements ClayDataSetDataProvider<UpgradeJava>
```

After
```
 public class UpgradeJavaClayDataSetDataProviderCheck implements FDSDataProvider<UpgradeJava>
```
Because of that, it will be validated by this if statement.

If that is the case, the following line is doing what?

`content = _updateImplementedClass(content);
`